lib/variable.js

```diff
@@ -91,30 +112,54 @@ module.exports = function getVariableAccessors (container, varName, defValue, ex
     asUrlObject: generateAccessor(require('./accessors/url-object')),
     asUrlString: generateAccessor(require('./accessors/url-string')),
 
+    /**
+     * Instructs env-var to first convert the value of the variable from base64
+     * to utf when reading it using a function such as asString()
+     */
     convertFromBase64: function () {
       isBase64 = true
 
       return accessors
     },
 
+    /**
+     * Set a default value for the variable
+     * @param {String} value
+     */
+    default: function (value) {
+      if (typeof value !== 'string') {
+        throw new EnvVarError('values passed to default() must be of type string')
```

Comment:
maybe you can check here for type:
string or number -> just print it
array and object -> json.stringify
other -> throw error

it will be more accurate; i mean you expect a number or an object but default it is a string

Reply:
Gotcha, it should try coerce the value to a string. Good suggestion 👍